packages/design-system/src/css/select.scss

```diff
@@ -165,4 +165,9 @@
 			height: 42px;
 		}
 	}
+	&--default {
+		.el-tag {
+			padding: 2px 12px;
```

Comment:
Can we use our spacing tokens (`--spacing-5xs` and `--spacing-xs`) instead of hard-coded pixel values?